src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MIN_VALUE = 1;
+    private static final int MAX_VALUE = 127;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomFigureIndex = random.nextInt(NUMBER_OF_FIGURES);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Colors randomColor = colorSupplier.getRandomColor();
+        int firstRandomNum = random.nextInt((MAX_VALUE - MIN_VALUE) + 1) + MIN_VALUE;
+        int secondRandomNum = random.nextInt((MAX_VALUE - MIN_VALUE) + 1) + MIN_VALUE
+                + random.nextInt(MAX_VALUE);
+        int thirdRandomNum = firstRandomNum + secondRandomNum;
```

Comment:
if it's a circle, we'll need only one of these `randomNum`, but we still will create two more. let's create private methods, shall we?